src/components/atoms/Checkbox/Checkbox.tsx

```diff
@@ -43,6 +45,8 @@ export const Checkbox: React.FC<CheckboxProps> = ({
         hidden
         type="checkbox"
         ref={forwardedRef}
+        id={id}
+        name={name ?? id}
```

Comment:
Reasoning was to avoid duplication and have more concise code.

Can move the defaulting behavior and since I think some of those labels used `id` just for the sake of the enclosing labels, can just stay with `name`

Reply:
The changes here can and should be reverted entirely IMO.

Reply:
fixed in ad646f2e6e4aa6b83b7f9dddd68cfcbcc0348b43